_data/banking.yml

```diff
@@ -574,6 +574,13 @@ websites:
       tfa: No
       lang: de
 
+    - name: ING-DiBa Deutschland
```

Comment:
Shouldn't it be `ING Deutschland`?

Reply:
Yes. We do have ING (Luxembourg), ING Australia but then IBG-DiBa Austria and ING-DiBa Deutschland.
Due to the alphabetical order restriction in Travis, we cannot rename ING-DiBa Deutschland to ING Deutschland without also renaming ING-DiBa Austria - which would be OK since _all_ product now go by "ING" rather than "ING DiBa" even though the bank in regulatory terms calls itself ING DiBa AG, at least in Frankfurt.
I'd go for renaming both banks in this PR before merge.